packages/apps/docs/src/docs/kadena/wallets/kadena-spirekey/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Kadena SpireKey User Guide
+description: Integrate Kadena SpireKey in your dApp
+menu: Kadena SpireKey
+label: Kadena SpireKey User Guide
+order: 2
+layout: full
+---
+
+# Kadena SpireKey User Guide
+
+---
+
+- [Kadena SpireKey User Guide](#kadena-spirekey-user-guide)
+  - [Short Introduction](#short-introduction)
+  - [Conceptual Overview](#conceptual-overview)
+    - [Create account in Kadena SpireKey](#create-account-in-kadena-spirekey)
+    - [Login with Kadena SpireKey](#login-with-kadena-spirekey)
+    - [Sign Transaction from dApp](#sign-transaction-from-dapp)
+  - [Specification](#specification)
+    - [OAuth authenticate](#oauth-authenticate)
+      - [Authentication request](#authentication-request)
+      - [Authentication Response](#authentication-response)
+    - [Kadena SpireKey Sign](#kadena-spirekey-sign)
+      - [Sign Request](#sign-request)
+      - [Sign Response](#sign-response)
+
+## Short Introduction
+
+In this document you will learn how to **connect** your dApp to Kadena SpireKey,
+**retrieve the available accounts** and **sign** with SpireKey Connect
+
+## Conceptual Overview
+
+From a dApp we want to access the accounts that are available on the Kadena
+SpireKey Wallet. With this information, we can then construct transactions.
+These transactions need to be signed by Kadena SpireKey
+
+We're all familiar with OAuth. OAuth enables the user to authenticate with a
+third-party authentication provider like Google, Facebook, Github and many more.
+
+![OAuth Flow](image.png)
+
+The similarities are:
+
+- the **dApp** is the **Client Application**
+- **Kadena SpireKey** works as a delegate for **Authorization Server** and in
+  turn acesses **WebAuthN** to grant access
+- and the **dApp** is hosted by the **Resource Server**
+
+Let's take a look in how the flow would be for a **dApp** and **Kadena
+Spirekey**
+
```

Comment:
```suggestion

```

I'm finding this hard to follow. See my somewhat mangled work-in-progress replacement text.